crossbeam-skiplist/src/base.rs

```diff
@@ -1401,6 +1401,12 @@ impl<'a, K: 'a, V: 'a> RefEntry<'a, K, V> {
         self.parent
     }
 
+    /// Releases the reference on the entry.
+    pub fn release(self, guard: &Guard) {
+        self.parent.check_guard(guard);
+        unsafe { self.node.decrement(guard) }
+    }
```

Comment:
Was this function modified in any way or just moved?

Reply:
It was just moved so that it can be called from `drop` without any trait bound.